### PR TITLE
Remove Timestamp padding from StoryPromo and StoryPromoList

### LINKED ISSUE #533: Remove Timestamp padding from StoryPromo and StoryPromoList
**Is your feature request related to a problem? Please describe.**
Following [this](https://github.com/bbc/psammead/issues/483) issue, we have to remove the padding from the Timestamp used in the `StoryPromo` and `StoryPromoList`.

Also, UX noticed that the Story Promo list item spacing is wrong. It should be:

- 0 - 600px:
  - Above: 8px - GEL_SPACING from @bbc/gel-foundations
  - Below: 16px - GEL_SPACING_DBL from @bbc/gel-foundations

**Describe the solution you'd like**
Add `padding={false}` prop to the `Timestamp` used in the `StoryPromo` and `StoryPromoList` stories.

Replace list item padding from `${GEL_SPACING_DBL} 0 ${GEL_SPACING}` to `${GEL_SPACING} 0 ${GEL_SPACING_DBL}`

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #533

**Overall change:**
Remove the padding from the `Timestamp` used in the `StoryPromo` and `StoryPromoList` stories
and swap the Story Promo list item spacing values for (0-600px) breakpoint.

**Code changes:**
- Add `padding={false}` prop to the Timestamp used in the `StoryPromo` story.
- Add `padding={false}` prop to the Timestamp used in the`StoryPromoList` story
- Replace `StyledListItem` `padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};` with `  padding: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};`

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-story-promo-list/CHANGELOG.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 0.1.0-alpha.2 | [PR#534](https://github.com/BBC-News/psammead/pull/XXX) Remove Timestamp padding. |
```
Comment (ghost) [inline packages/components/psammead-story-promo-list/CHANGELOG.md]:
```suggestion
| 0.1.0-alpha.2 | [PR#534](https://github.com/BBC-News/psammead/pull/534) Remove Timestamp padding. |
```

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 0.1.5   | [PR#534](https://github.com/BBC-News/psammead/pull/XXX) Remove Timestamp padding. |
```
Comment (ghost) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 0.1.5   | [PR#534](https://github.com/BBC-News/psammead/pull/534) Remove Timestamp padding. |
```

Comment (DenisHdz) [reply packages/components/psammead-story-promo/CHANGELOG.md]: Ups, CTRL+D didn't work 😅

**Review by ghost — DISMISSED** [review]
LGTM 👍

**Review by andrew-nowak — DISMISSED** [review]
:+1:

**Review by 12 — DISMISSED** [review]
LGTM - _(but needs conflicts resolving first)_

**Review by ghost — DISMISSED** [review]
👍

**Review by 12 — DISMISSED** [review]
LGTM 👍

**Comment by david-boydell** [comment]
Hiya,

I think the StoryPromoList needs to use the latest `psammead-story-promo` version.

**Review by ghost — DISMISSED** [review]
👍

**Review by dr3 — DISMISSED** [review]
In future we should ensure, via unit testing, that this prop is being set. Nowhere in the tests or snaps can I see evidence this prop is being passed.

For now though, LGTM

**Comment by david-boydell** [comment]
Good stuff, this is RFM.

**Comment by dr3** [comment]
Needs a version bump

**Review by ghost — APPROVED** [review]
LGTM

**Review by dr3 — APPROVED** [review]

**Comment by david-boydell** [comment]
This is ready for merge.

**Pull request merged**